src/viewport/index.tsx

```diff
@@ -265,6 +267,8 @@ class Viewport extends React.Component<
                 simulariumController.initializeTrajectoryFile();
             }
         };
+        simulariumController.startRecording = this.startRecording.bind(this);
+        simulariumController.stopRecording = this.stopRecording.bind(this);
```

Comment:
bind the viewport methods (which have the right context) to the controller methods for the front end to call